File: 01/cpp/startpoint.cpp
#include <fstream>
#include <iostream>
#include <vector>
#include <set>

using namespace std;

bool has_a_sum(const std::vector<int> & arr, const int k){
    return false;
}

int main(){

    // Read test case
    ifstream inFile;
    inFile.open("../testcase.txt");

    if(!inFile){
        cerr << "Unable to open the file" << endl;
        return -1;
    }

    // Buffer variables to read test case
    int T,N,K;
    inFile >> T;

    // Loop over test cases
    for(int t = 1; t <= T; t++){

        // Read length of array N and sum K
        inFile >> N >> K;

        // Fill array
        vector<int> arr(N);
        for(int n = 0; n < N; n++){
            inFile >> arr[n];
        }

        // Print solution
        cout << "Case #" << t << ": " << has_a_sum(arr, K) << endl;
    }

    // Close file
    inFile.close();

    return 0;
}

File: 01/python/startpoint.py
def has_a_sum(arr, K):
	pass
	
if __name__ == '__main__':

	# Read test file
	f = open("../testcase.txt", "r")

	# Number of test cases
	T = int(f.readline())

	# Loop over test cases
	for __ in xrange(T):

		# Read length of array N and sum K
		N,K = map(int, f.readline().split())

		# Fill array
		arr = map(int, f.readline().split())

		# Print solution
		print("Case #%d: %s" % (__+1, has_a_sum(arr, K)))

	# Close file
	f.close()